tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "outDir": "./",
+    "removeComments": false,
+    "noImplicitAny": false
+  },
+  "exclude": [
+      "node_modules",
+      "./typescript/types.ts"
```

Comment:
This file isn't excluded by the compiler. Typescript also doesn't support globs yet.